hawkrest/__init__.py

```diff
@@ -23,10 +23,44 @@
 default_message_expiration = 60
 
 
+def lookup_credentials(cr_id):
+    if cr_id not in settings.HAWK_CREDENTIALS:
+        raise LookupError('No Hawk ID of {id}'.format(id=cr_id))
+    return settings.HAWK_CREDENTIALS[cr_id]
+
+
+def lookup_user(request, credentials):
+    return HawkAuthenticatedUser(), None
+
+
+settings_user_lookup = getattr(
+    settings,
+    'HAWK_USER_LOOKUP',
+    None)
+user_lookup = (
+    import_string(settings_user_lookup) if settings_user_lookup
+    else lookup_user)
+
+settings_cred_lookup = getattr(
+    settings,
+    'HAWK_CREDENTIALS_LOOKUP',
+    None)
+credentials_lookup = (
+    import_string(settings_cred_lookup) if settings_cred_lookup
```

Comment:
If this gets moved moved into the default `HawkAuthentication._credentials_lookup` method or some utility function like `get_credentials_lookup()` then I think testing would be easier. You could inject a mock instead of rely on `reload_module()`.